client/components/Reserve.jsx

```diff
@@ -143,3 +140,19 @@ const FindWrap = styled.div`
     background-color: #e45962;
   }
 `;
+
+const TimeSlot = styled.div`
+  color: white;
+  background-color: #da3743;
+  border-radius: 5%;
+  width: 80px;
+  height: 40px;
+  margin: 8px;
+  text-align: center;
+  display: table-cell;
+  vertical-align: middle;
+  &:hover {
+    cursor: pointer;
+    background-color: #e45962;
+  }
```

Comment:
not 100% certain, but possibly a semicolon at the end after the closing bracket?